setup.py

```diff
@@ -291,10 +309,20 @@ def run(self):
     elif sys.platform == 'darwin':
         ext_modules.append(Extension('borg.platform_darwin', [platform_darwin_source]))
 
+        
+        
+def parse(root, describe_command=None):
+    return subprocess.check_output("git describe --tags --long").decode().strip()
+
+parsefunction = None
+if sys.platform == 'win32':
+    parsefunction = parse
+
 setup(
     name='borgbackup',
     use_scm_version={
         'write_to': 'borg/_version.py',
+        'parse': parsefunction,
```

Comment:
hmm, is this something that needs fixing in setuptools-scm, so it works by default on windows?